cookbooks/aws-parallelcluster-environment/recipes/install/cfn_bootstrap.rb

```diff
@@ -15,7 +15,7 @@
 virtualenv_name = 'cfn_bootstrap_virtualenv'
 pyenv_root = node['cluster']['system_pyenv_root']
 # FIXME: Python Version cfn_bootstrap_virtualenv due to a bug with cfn-hup
```

Comment:
TODO; to be removed and merge cfn_bootstrap_virtualenv into one of the other virtual env (node or cookbook)

Reply:
I created an internal task for this.